### PR TITLE
MRG: Allow maxshield silently

### PR DESCRIPTION
**Pull request by Eric89GXL**
For people who record with MaxShield on and plan on running `maxwell_filter` on the data, it would be nice to be able to silently load data. Now `allow_maxshield='yes'` allows us to do that.

### CODE REVIEW

File: mne/io/fiff/raw.py [hunk 1]
```diff
@@ -182,7 +184,9 @@ def _read_raw_file(self, fname, allow_maxshield, preload, compensation,
                         raise ValueError('No raw data in %s' % fname)
                     elif allow_maxshield:
                         info['maxshield'] = True
-                        warn(msg)
+                        if not (isinstance(allow_maxshield, string_types) and
+                                allow_maxshield.startswith('y')):
```
Comment (agramfort) [inline mne/io/fiff/raw.py]: allow_maxshield == "yes"

???

Comment (Eric89GXL) [inline mne/io/fiff/raw.py]: Just allowing for expert extra laziness :) I'll change it.

**Comment by Eric89GXL** [comment]
@agramfort fixed, okay for merge once the CIs are happy?

**Comment by Eric89GXL** [comment]
(got okay from @agramfort while talking to him off-GH)

**Pull request merged**